rides_report.rb

```diff
@@ -0,0 +1,96 @@
+# A program that uses rideshare information to provide aggregated metrics
+# 8/11/19
+# Hallie Johnson
+
+require 'colorize'
+
+drivers = [
+  {
+    id: "DR0001",
+    ride_date: ["3rd Feb 2016", "3rd Feb 2016", "5th Feb 2016"],
+    cost: [10, 30, 45],
+    rider: ["RD0003", "RD0015", "RD0003"],
+    rating: [3, 4, 2]
```

Comment:
The way you have this organized currently makes it easy to iterate, but hard for humans to read. Just as correct of answers, but that is a trade-off worth being aware of.